php/Dockerfile-7.1

```diff
@@ -16,7 +16,11 @@ RUN apt-get update -q -y \
         curl \
         acl \
         sudo \
+# Remove expired Let's Encrypt certificate
+    && rm /usr/share/ca-certificates/mozilla/DST_Root_CA_X3.crt \
```

Comment:
I suggest not to remove the certificate, but to actually use the blacklisting method.
On CentOS it's /etc/pki/ca-trust/source/blacklist/dst_root_ca_x3.crt (you have to copy the certificate file there), but on Ubuntu we must locate that path.